src/Microsoft.DotNet.ProjectJsonMigration/ProjectExtensions.cs

```diff
@@ -13,18 +13,27 @@ internal static class ProjectExtensions
     {
         public static ProjectType GetProjectType(this Project project)
         {
-            var projectType = ProjectType.Console;
             if (project.IsWebProject())
             {
-                projectType = ProjectType.Web;
+                return ProjectType.Web;
```

Comment:
I don't like this multiple return invocation pattern. Can you please revert this to the way it was before. With the assignment of the variable and a single return?